### PR TITLE
feat: New fasthttp integration

### PR DESCRIPTION
**Pull request by kamilogorek**
TODO: Add to Docs once merged.

### CODE REVIEW

File: example/fasthttp/main.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
```
Comment (bruno-garcia) [inline example/fasthttp/main.go]: nit: This empty line is some Go hypster thing?

File: fasthttp/sentryfasthttp.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
+package sentryhttp
+
+import (
+  "context"
+  "fmt"
+  "net"
+  "strings"
+  "time"
+
```
Comment (bruno-garcia) [inline fasthttp/sentryfasthttp.go]: yeah I see

File: fasthttp/sentryfasthttp.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  r := sentry.Request{}
+
+  r.Method = string(ctx.Method())
+  uri := ctx.URI()
+  r.URL = fmt.Sprintf("%s://%s%s", uri.Scheme(), uri.Host(), uri.Path())
+
+  // Headers
+  headers := make(map[string]string)
+  ctx.Request.Header.VisitAll(func(key, value []byte) {
+    headers[string(key)] = string(value)
```
Comment (bruno-garcia) [inline fasthttp/sentryfasthttp.go]: Should we drop `Auth` header?

File: example/fasthttp/main.go (hunk 1 above)
Comment (kamilogorek) [reply example/fasthttp/main.go]: `gofmt` itself adds new line between native and external imports.

File: fasthttp/sentryfasthttp.go (hunk 3 above)
Comment (kamilogorek) [reply fasthttp/sentryfasthttp.go]: Do we actually it like this anywhere? I know that we don't in sentry/node for sure.

File: fasthttp/README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+go get github.com/getsentry/sentry-go/fasthttp
+```
+
+```go
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/getsentry/sentry-go"
+	sentryfasthttp "github.com/getsentry/sentry-go/fasthttp"
```
Comment (YashishDua) [inline fasthttp/README.md]: If not imported as `sentryfasthttp`, it should primitively be `fasthttp`.

File: fasthttp/README.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+fastHTTPHandler := func(ctx *fasthttp.RequestCtx) {
+	switch string(ctx.Path()) {
+	case "/foo":
+		fooHandler(ctx)
+	default:
+		defaultHandler(ctx)
+	}
+}
+
+fmt.Println("Listening and serving HTTP on :3000")
```
Comment (YashishDua) [inline fasthttp/README.md]: This should come after we responded to the error from `ListenAndServe`.

File: fasthttp/README.md (hunk 4 above)
Comment (kamilogorek) [reply fasthttp/README.md]: `fasthttp` is the name of the package we wrap, so I cannot use the same name here. `goimports` takes this name automatically from the package's top declaration.

File: fasthttp/README.md (hunk 5 above)
Comment (kamilogorek) [reply fasthttp/README.md]: 👍

File: fasthttp/sentryfasthttp.go [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  // Timeout for the event delivery requests.
+  Timeout time.Duration
+}
+
+// New returns a struct that provides Handle method
+// that satisfy fasthttp.RequestHandler interface.
+func New(options Options) *Handler {
+  handler := Handler{
+    repanic:         false,
+    timeout:         time.Second * 2,
```
[the remark this answers is not in the document]
Comment (kamilogorek) [reply fasthttp/sentryfasthttp.go]: Good catch! Missed that one, thanks!

**Pull request merged**